### PR TITLE
Don't uninstall before installing in `flutter run`

### PR DESCRIPTION
**Pull request by tvolkert**

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/install.dart [hunk 1]
```diff
@@ -33,11 +33,11 @@ class InstallCommand extends FlutterCommand {
   }
 }
 
-bool installApp(Device device, ApplicationPackage package) {
+bool installApp(Device device, ApplicationPackage package, {bool uninstall: true}) {
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/install.dart]: nit: spaces around names arguments

**Comment by Hixie** [comment]
LGTM

Did you figure out the weirdness?

**Comment by devoncarew** [comment]
lgtm as well; thanks!

**Pull request merged**